### PR TITLE
fix: add favicon, title, Google analytics

### PR DESCRIPTION
**Pull request by VinneyJ**
1. Add CivicSignal favicon support
2. Change title from Media Cloud to Civic Signal
3. Add google analytics.

### CODE REVIEW

File: server/templates/index.html [hunk 1]
```diff
@@ -7,11 +7,27 @@
   <meta name="viewport"
     content="width=device-width, initial-scale=1, user-scalable=0, maximum-scale=1, minimum-scale=1"
   >
-  <link rel="icon" type="image/png" href="{{ url_for('static', filename='favicon.png') }}" />
+  <link rel="apple-touch-icon" sizes="180x180" href={{ url_for('static', filename='apple-touch-icon.png') }}" />
+  <link rel="icon" type="image/png" sizes="32x32" href="{{ url_for('static', filename='favicon-32x32.png') }}" />
+  <link rel="icon" type="image/png" sizes="16x16" href="{{ url_for('static', filename='favicon-16x16.png') }}" />
+  <link rel="manifest" href="{{ url_for('static', filename='site.webmanifest') }}" />
+  <link rel="shortcut icon" href="{{ url_for('static', filename='favicon.ico') }}" />
+  <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#022069" />
```
Comment (kilemensi) [inline server/templates/index.html:15]: In addition to the files you've included in this PR, the zipped file should also contain:

1. browserconfig.xml
2. safari-pinned-tab.svg
3. site.webmanifest
4. mstile-150x150.png

Please add them to the static folder as well.

**Review by kilemensi — APPROVED** [review]
🚀 ... that's more like it. Close #14

File: server/templates/index.html (hunk 1 above)
Comment (VinneyJ) [reply server/templates/index.html:15]: Cool! added.

Comment (kilemensi) [inline server/templates/index.html:15]:
```suggestion
  <link rel="mask-icon" href="{{ url_for('static', filename='safari-pinned-tab.svg') }}" color="#022069" />
```

**Review by kelvinkipruto — APPROVED** [review]

**Pull request merged**